x-pack/plugins/security_solution/server/lib/detection_engine/rule_actions/legacy_saved_object_mappings.ts

```diff
@@ -30,10 +30,14 @@ const legacyRuleActionsSavedObjectMappings: SavedObjectsType['mappings'] = {
     },
     actions: {
       properties: {
+        actionRef: {
+          type: 'keyword',
+        },
         group: {
           type: 'keyword',
         },
         id: {
+          // "actions.id" is no longer used since the saved object references and "actionRef" was introduced. It is still here for legacy reasons such as migrations.
```

Comment:
Actually I have to keep it because older data will need to be migrated and we cannot remove the mapping as far as I know. I don't know if the mappings are changed _before_ the migration but I assume it would have to be since I already am adding the `actionRef` above. So if the mappings change _before_ the migration (which I'm pretty sure it does and would need to), and I remove this here I don't know what the effect is going to be. I might still be able to access it without it but I figured it was best to leave it alone for now?